src/main/java/core/basesyntax/StorageImpl.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax;
+
+public class StorageImpl<K, V> implements Storage<K, V> {
+    static final int MAX_NUMBER = 10;
```

Comment:
Let's be clear with naming and access modifiers
```suggestion
    private static final int MAX_SIZE = 10;
```